jenkins-scripts/tools/git_delete_closed_hg_branches.bash

```diff
@@ -9,7 +9,13 @@ echo =========== Done Cloning HG Repository
 echo
 echo =========== Identify Closed HG Branches
 CLOSED_BRANCHES=$(\
-  hg -R $HG_TMP log -r "closed()" --template '{branch}\n' | sort)
+  hg branches -c | grep '(closed)$' | sed -e 's@ *[0-9]*[0-9]:.*@@')
```

Comment:
if the intention of that sed is to remove every but the first *column*, we could use `awk '{ print $1 }' which seems safer. No big deal anyway.
```suggestion
  hg branches -c | grep '(closed)$' | awk '{ print $1 }')
```
`

Reply:
yeah, that `awk` script is cleaner; I think I'm just haunted by the fact that [mercurial branches can have whitespace characters](http://mercurial.808500.n3.nabble.com/Escaping-characters-for-hg-update-td3994992.html) so I made a partial attempt to supporting that.

now that I think about it, the proper way would be to extract the commit hash from the `hg branches` call and use `hg log -r ${changeset} --template '"{branch}"'` and do escaping properly. I'm not sure we have any branch names with spaces though, what an evil thing that would be